exonum/src/events/noise/wrappers/sodium_wrapper/resolver.rs

```diff
@@ -84,10 +84,27 @@ impl Default for SodiumRandom {
     }
 }
 
-impl Random for SodiumRandom {
+impl Random for SodiumRandom {}
+
+impl CryptoRng for SodiumRandom {}
+
+impl RngCore for SodiumRandom {
+    fn next_u32(&mut self) -> u32 {
```

Comment:
Maybe we should add a comment that this method is unused and `uninplemented` here is safe.

Reply:
Or just use `unreachable`.